backend/docs/installation.md

```diff
@@ -1,5 +1,30 @@
 # Installation
 
+The recommended setup is to use the dev container rather than installing things to your local environment. The dev container setup will ensure you have the same setup as others using docker.
+
+## Dev Container
+
+There are two ways to run dev containers, in browser via Github codespaces (easiest) or locally via docker and vscode.
+
+### Github Codespaces
+
+1. Head over to the repo on Github: https://github.com/codeforboston/urban-league-heat-pump-accelerator
+2. Locate the Code dropdown, open the Codespaces tab
+   1. Hit the + to create a new Codespace. This will open-up vscode in browser and you'll be able to start contributing!
+
+### Local Dev Containers
+
+1. Clone the project
+    ```bash
+   git clone git@github.com:codeforboston/urban-league-heat-pump-accelerator.git
```

Comment:
The "local install" instructions below say to fork the project, but it seems like the instructions in this section say just to work with the Code for Boston repo directly. If that's now the intent, should this also be updated in the local install section, so that the two sections match? (I checked the project README just in case it expressed opinions on forking vs not forking, but it doesn't get into that level of detail: https://github.com/codeforboston/urban-league-heat-pump-accelerator.)

Reply:
yeah I'll make them match!